src/helpers/build.js

```diff
@@ -47,6 +48,44 @@ const copySupportFiles = folder => {
   } else {
     shell.cp('-r', path.join(process.cwd(), 'node_modules/wpe-lightning-sdk/support/*'), folder)
   }
+
+  // since esbuild has a different ouput over rollup
+  // we need to copy a different startApp file
+  if (process.env.LNG_BUNDLER === 'esbuild') {
+    shell.cp(
+      '-r',
+      path.join(__dirname, '../../fixtures/support/startApp.js'),
+      path.join(folder, 'startApp.js')
+    )
+  }
+
+  const command = process.argv.pop()
+
+  // if live reload is enabled we write the client WebSocket logic
+  // to index.html
+  if (process.env.LNG_LIVE_RELOAD === 'true' && command === 'dev') {
+    const port = process.env.LNG_LIVE_RELOAD_PORT || 8991
+    const file = path.join(folder, 'index.html')
+    const data = fs.readFileSync(file, { encoding: 'utf8' })
+    const wsData = `
+      <script>
+        const socket = new WebSocket('ws://localhost:${port}');
+        socket.addEventListener('open', ()=>{
+          console.log('WebSocket connection succesfully opened - live reload enabled');
+        });
+        socket.addEventListener('close', ()=>{
+          console.log('WebSocket connection closed - live reload disabled');
+        });
+        socket.addEventListener('message', (event)=>{
+          if(event.data === "reload"){
+            document.location.reload();
+          }
+        });        
+      </script>
+    </body>`
```

Comment:
should this be es5 compatible code? in case someone wants to do live reload on an es5 compatible browser?